tests/cbmc/proofs/s2n_blob_zero/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+cbmcflags: "--bounds-check;--conversion-check;--div-by-zero-check;--enum-range-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--pointer-primitive-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwind;1;--unwinding-assertions;--object-bits;8"
```

Comment:
I think we are autogenerating these files. Use the same form as the other autogenerated ones.

Reply:
Done.